src/screencast/pipewire_screencast.c

```diff
@@ -26,6 +26,38 @@ static void writeFrameData(void *pwFramePointer, void *wlrFramePointer,
 	return;
 }
 
+static inline struct spa_pod *build_format(struct spa_pod_builder *b, uint32_t format, uint32_t format_without_alpha,
```

Comment:
Don't use `inline`, the compiler will automatically inline functions if it's worth it.

Reply:
done